builder/ecs/access_config.go

```diff
@@ -28,6 +28,10 @@ type AlicloudAccessConfig struct {
 	// Alicloud region must be provided unless `profile` is set, but it can
 	// also be sourced from the `ALICLOUD_REGION` environment variable.
 	AlicloudRegion string `mapstructure:"region" required:"true"`
+	// Alicloud RamRole must be provided for EcsRamRole mode
+	AlicloudRamRole string `mapstructure:"ram_role_name" required:"false"`
+	// Alicloud authentication mode must be provided
```

Comment:
Is there some additional documentation needed here? It is not clear to me why mode must be provided and what are the allowed values. 

It appears that mode is a concept used internally within the code for auth related things but I'm not familiar with it. As a first time user of Alicloud I find this confusing. Have you considered adding an example template configuration to illustrate the usage, as well?